scripts/actions/add-files-to-translation-queue.js

```diff
@@ -84,7 +85,20 @@ const main = async () => {
 
   await saveToTranslationQueue(key, 'set locales = :slugs', { ':slugs': data });
 
-  process.exit(0);
+  // If we are in a CI environment, return a success code
+  if (process.env.CI) {
```

Comment:
Github Actions automatically set this to `true` and it doesn't look like you can override it. 
You can use `require.main === module` instead, it will evaluate to `true` if the file is executed by Node and `false` if the file is imported by another module. https://nodejs.org/docs/latest/api/modules.html#modules_accessing_the_main_module